gen-bundle-image.sh

```diff
@@ -11,6 +11,12 @@ function set_bundle_variables {
    libvirt_bundle=crc_libvirt_${version}_amd64.crcbundle
    hyperv_bundle=crc_hyperv_${version}_amd64.crcbundle
 
+   if [[ ${preset} = "okd" ]]; then
+       vfkit_bundle=crc_okd_vfkit_${version}_amd64.crcbundle
+       libvirt_bundle=crc_okd_libvirt_${version}_amd64.crcbundle
+       hyperv_bundle=crc_okd_hyperv_${version}_amd64.crcbundle
+   fi
+
```

Comment:
Alternatively, could become something like:
```
if [[ ${preset} = "podman" ]]; then
    vfkit_bundle_arm64=crc_podman_vfkit_${version}_arm64.crcbundle
fi
if [[ ${preset} = "openshift" ]]; then
    preset=""
else
    preset="_${preset}"
fi

vfkit_bundle=crc${preset}_vfkit_${version}_amd64.crcbundle
libvirt_bundle=crc${preset}_libvirt_${version}_amd64.crcbundle
hyperv_bundle=crc${preset}_hyperv_${version}_amd64.crcbundle
```

Reply:
@cfergeau I will use this suggestion for another refactor where now we also release OCP for M1 so include vfkit arm64 bundle for ocp also.